ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -71,10 +75,12 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
   @task
   @waitFor
   *onFeatureConfirm() {
+    // must return null instead of root for non managed cluster.
+    const namespace = this.args.isHvdManaged ? 'admin' : null;
```

Comment:
The behavior is still that it can only be enabled in root or admin, right? Not other child namespaces? Might be good to add that to the comment

Reply:
Yep, you're correct. And I'll amend the comment to include, good idea.